scripts/deps.sh

```diff
@@ -85,6 +86,9 @@ restore_go_dep() {
     dep)
       (cd "${__proj_dir}" && dep ensure -v -vendor-only)
       ;;
+    gomod)
+      (cd "${__proj_dir}" && go mod vendor)
```

Comment:
please port that change to pluginsync (or make a JIRA issue)

Reply:
https://swicloud.atlassian.net/browse/AO-17506